### ISSUE TITLE
[GitHub Codespaces] There is no prompt "Cannot edit in read-only editor" when editing a container file

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-docker#2494]
**Build Version:** 20201114.1

**Repro Steps:**
1. Open a container file from container view.
2. Edit it and check the result.

**Expect:**
There is a prompt "Cannot edit in read-only editor".
![image](https://user-images.githubusercontent.com/45614674/99227456-3b148180-2826-11eb-93b6-fc73eee4eb53.png)

**Actual:**
There is no prompt "Cannot edit in read-only editor".
![image](https://user-images.githubusercontent.com/45614674/99227503-4b2c6100-2826-11eb-8683-d23e6bc7461f.png)

**Comment by bwateratmsft** [comment]
I haven't been able to reproduce, the "Cannot edit in read-only editor" tag appears for me. What OS did you try on and is this VSCode Insiders?

**Comment by bwateratmsft** [comment]
I missed that this was in Codespaces, I'll try there.

**Comment by v-ruizh** [comment]
@bwateratmsft Sorry for miss the information. This issue only reproduces with remote/browser scenario. I guess you may repro it with local scenarios. You can reproduce this issue with below steps.

1.	Navigate to the source repo on GitHub.
2.	Click the “Code” button and choose “Open with Codespaces”.
3.	Click the “+ New Codespace” button.
4.	Once the GHCS is loaded, install the Docker extension and reproduce this issue.

Please feel free to let me know if you have any concern.

**Comment by bwateratmsft** [comment]
I did try it on Codespaces but was not able to reproduce the issue. Does it happen consistently?

**Comment by v-ruizh** [comment]
@bwateratmsft This issue happens consistently. You can try to add something to reproduce this issue. This issue does not reproduce if you try to delete something.

**Comment by xkungfu** [comment]
same issue.
why closed?

**Comment by v-ruizh** [comment]
@bwateratmsft This issue also reproduces, reopen it. Could you help to take a look?

**Comment by bwateratmsft** [comment]
Ok, I was able to repro but only in the web editor and only when trying to add--it still shows when I try to delete (i.e. with Backspace). This would be an issue with VSCode and/or the web editor, not with the Docker extension. I opened https://github.com/microsoft/vscode/issues/113854 for that but since this isn't very serious I'll close this issue. (Also, we plan on making the files editable anyway with #2465)
